template.tpl

```diff
@@ -34,7 +34,7 @@ ___TEMPLATE_PARAMETERS___
     "macrosInSelect": false,
     "selectItems": [
       {
-        "displayValue": "Init",
```

Comment:
this is actually the method name, so I would give it a display value other than its name.
In fact I think we should even remove the capitals and have them camel cased to avoid unnecessary confusion.

Reply:
Yes, this was me making it nicer, I should have put that in another PR; nothing related to this,.